umh.docs.umh.app/content/en/docs/getstarted/installation.md

```diff
@@ -2,32 +2,32 @@
 title: "1. Installation"
 menuTitle: "1. Installation"
 description: |
-   Install the United Manufacturing Hub together with all required tools on a Linux Operating System.
+  Install the United Manufacturing Hub together with all required tools on a Linux Operating System.
 weight: 1000
 edition: community
 ---
 
-The United Manufacturing Hub (UMH) can be deployed on various external devices, 
-including edge devices and virtual machines (VMs). For initial installations 
+The United Manufacturing Hub (UMH) can be deployed on various external devices,
+including edge devices and virtual machines (VMs). For initial installations
 or for development purposes, it is recommended to use a VM.
 
 ## Software Requirements
 
 The UMH installation requires one of the following Operating System on your server:
 
-- Flatcar version current-2023 or higher (3510.3.1). 
-It is recommended that you have full control over the operating system. To install Flatcar on your server, 
-follow [this guide](/docs/production-guide/installation/flatcar-installation/). <!-- This article needs to be merged together https://umh.docs.umh.app/docs/production-guide/installation/flatcar-installation-virtual-machine/ https://learn.umh.app/course/flatcar-installation-on-proxmox/ -->
-- Red Hat Enterprise Linux (RHEL) 9.0 and higher. Recommended when you can choose 
-only out of a small amount of potential Operating System in your large enterprise
-- Community Supported: Ubuntu 22.04.4 LTS. This approach is useful 
-when you're for example trying to install the UMH on a cloud instance like AWS EC2 
-and struggle to install Flatcar or RHEL there.
-
-While UMH is optimized for RHEL and Flatcar, it can theoretically run on other Linux distributions. 
-However, support is not guaranteed. 
-For Windows, you could try running one of the above described Operating Systems in a VM (e.g., Hyper-V). 
-If you experiment with other systems, we encourage sharing your experiences 
+- Flatcar version current-2023 or higher (3510.3.1).
+  It is recommended that you have full control over the operating system. To install Flatcar on your server,
+  follow [this guide](/docs/production-guide/installation/flatcar-installation/). <!-- This article needs to be merged together https://umh.docs.umh.app/docs/production-guide/installation/flatcar-installation-virtual-machine/ https://learn.umh.app/course/flatcar-installation-on-proxmox/ -->
+- Red Hat Enterprise Linux (RHEL) 9.0 and higher. Recommended when you can choose
+  only out of a small amount of potential Operating System in your large enterprise
```

Comment:
Add a . at the end of the sentence. 
The sentece sounds weird, just say: ... `out of a small amount of potential Operating Systems.`

Reply:
This was already like that, didn't touch it at all, it just got formatted.

Reply:
still a valid point :)